### ISSUE TITLE
allowframebreaks introduces vertical space before first frame

### ISSUE THREAD

**Issue by josephwright** [josephwright/beamer#325]
Originally reported by: **ignasi (Bitbucket: [ignasi](https://bitbucket.org/ignasi), GitHub: [ignasi](https://github.com/ignasi))**

----------------------------------------

When option `allowframebreaks` and `\framebreak` command are used, the first frame from the series shows a different vertical space between title and text than all other frames. This happens with verticaly centered or top aligned frames.

Next code shows the problem. Resulting pdf file is attached.

```
#!latex

\documentclass{beamer}
\usepackage{lipsum}
\begin{document}

\begin{frame}[allowframebreaks]{top aligned frame}

\lipsum[2]
\framebreak

\lipsum[2]
\framebreak

\lipsum[2]
\framebreak

\lipsum[2]
\end{frame}
\end{document}
```

----------------------------------------
- Bitbucket: https://bitbucket.org/rivanvx/beamer/issue/325

**Comment by josephwright** [comment]
*Original comment by* **stefanct (Bitbucket: [stefanct](https://bitbucket.org/stefanct), GitHub: [stefanct](https://github.com/stefanct))**:

----------------------------------------

I can confirm this with version 3.33. While `allowframebreaks` might be evil in many cases I use it when creating dedicated slides with solutions for exercises that I can switch on and of and it looks quite bad if the exercise block is not on the same height as the solution block that is put on the follow-up slide.

**Comment by JTatGitHub** [comment]
1. This bug does not seem to be fixed in v3.50. I would really like to see it fixed.
2. The bug does not only show up when the `\framebreak` command is issued by the user, but also when beamer breaks the frame automatically.
3. The problem is not that the vertical spacing on the first slide is too big, but that it is too small on the following slides.

**Comment by louisstuart96** [comment]
The same problem happens in this MWE:
```latex
\frame[t]{%
{Test frame}
\begin{itemize}[<only@+>]
  \item First slide has a larger vmargin?
  \item Second
  \item Third
\end{itemize}
}
```
